js/index.js

```diff
@@ -1 +1,76 @@
 // Your code goes here
+
+
+//Event 1
+const busPicture = document.getElementById('bus');
+busPicture.addEventListener('mouseenter', () => {
+    busPicture.setAttribute('src', 'img/Game-Crazy-Bus.jpg');
+    busPicture.style.maxWidth = '100%';
+});
+//Event 2
+busPicture.addEventListener('mouseleave', () => {
+    busPicture.setAttribute('src', 'img/fun-bus.jpg');
+});
+//Event 3
+const logo = document.querySelector('.logo-heading');
+logo.addEventListener('dblclick', () => {
+    alert `Ahhh! you double clicked the logo! - Easter Egg`;
+});
+//Event 4
+const firstPic = document.getElementById('map-picture');
+firstPic.addEventListener('click', () => {
+    firstPic.style.display = 'none';
+})
+//Event 5
+const secondPic = document.getElementById('boat');
+secondPic.addEventListener('mouseover', () => {
+    secondPic.style.border = '2px dashed dodgerblue';
+})
+
+//Event 6
+const pTags = document.querySelectorAll('p');
+for(let i = 0; i < pTags.length; i++){
+    pTags[i].addEventListener('wheel', (event) => {
+        event.target.style.border = '2px dashed magenta';
+    })
+}
+
+//Event 7
+const html = document.querySelector('html');
+html.addEventListener('keydown', (event) => {
```

Comment:
You could accompany the keydown with a keyup and have some pretty cool animations.